neqo-transport/src/recovery/mod.rs

```diff
@@ -815,17 +815,36 @@ impl LossRecovery {
     /// When it has, mark packets as "lost" for the purposes of having frames
     /// regenerated in subsequent packets.  The packets aren't truly lost, so
     /// we have to clone the `SentPacket` instance.
-    fn maybe_fire_pto(&mut self, rtt: &RttEstimate, now: Instant, lost: &mut Vec<SentPacket>) {
+    fn maybe_fire_pto(&mut self, primary_path: &PathRef, now: Instant, lost: &mut Vec<SentPacket>) {
         let mut pto_space = None;
         // The spaces in which we will allow probing.
         let mut allow_probes = PacketNumberSpaceSet::default();
         for pn_space in PacketNumberSpace::iter() {
-            if let Some(t) = self.pto_time(rtt, pn_space) {
+            if let Some(t) = self.pto_time(primary_path.borrow().rtt(), pn_space) {
```

Comment:
To reduce indentation, each of these `if`s in the `for` loop could as well be an `else { continue }`.

Happy to do that in a follow-up. Keeping as is for now to keep the diff small.